src/bdscript/addCmdReactions.md

```diff
@@ -1,9 +1,9 @@
 # $addCmdReactions
 Adds reactions to the message that triggered the command.
 
-## Usage
+## Syntax
 ```
-$addCmdReactions[emojis;...]
+$addCmdReactions[emojis]
```

Comment:
This ellipsis is important. It quickly shows that you can repeat the last argument.
```suggestion
$addCmdReactions[emojis;...]
```
At the least, this is the reason it's there in the first place.